r/R/dataset-write.R

```diff
@@ -53,6 +53,34 @@
 #' - `codec`: A [Codec] which will be used to compress body buffers of written
 #'   files. Default (NULL) will not compress body buffers.
 #' @return The input `dataset`, invisibly
+#' @examples
+#' dout1 <- paste0(tempdir(), "/direct")
```

Comment:
I suspect we will want to wrap this in `\donttest{}` since there is a possibility this might not fully function on some platforms. Ideally we wouldn't do that, but I suspect we might have to here.